Framework/HistogramData/src/HistogramMath.cpp

```diff
@@ -0,0 +1,107 @@
+#include "MantidHistogramData/Histogram.h"
+#include "MantidHistogramData/HistogramMath.h"
+
+#include <algorithm>
+#include <cmath>
+#include <stdexcept>
+
+namespace Mantid {
+namespace HistogramData {
+
+/** Scales data in histogram by constant (non-negative) factor.
+ *
+ * Uncertainties are scaled by the same factor, such that the *relative*
+ * uncertainties remain unchanged. */
+Histogram &operator*=(Histogram &histogram, const double factor) {
+  if (factor < 0.0 || !std::isfinite(factor))
+    throw std::runtime_error("Invalid operation: Cannot scale Histogram by "
+                             "negative or infinite factor");
+  histogram.mutableY() *= factor;
+  histogram.mutableE() *= factor;
+  return histogram;
+}
+
+/** Divides data in histogram by constant (positive) factor.
+ *
+ * Uncertainties are divided by the same factor, such that the *relative*
+ * uncertainties remain unchanged. */
+Histogram &operator/=(Histogram &histogram, const double factor) {
+  return histogram *= 1.0 / factor;
+}
+
+/** Scales data in histogram by constant (non-negative) factor.
+ *
+ * Uncertainties are scaled by the same factor, such that the *relative*
+ *uncertainties remain unchanged. */
+Histogram operator*(Histogram histogram, const double factor) {
+  return histogram *= factor;
+}
+
+/** Scales data in histogram by constant (non-negative) factor.
+ *
+ * Uncertainties are scaled by the same factor, such that the *relative*
+ *uncertainties remain unchanged. */
+Histogram operator*(const double factor, Histogram histogram) {
+  return histogram *= factor;
+}
+
+/** Dividies data in histogram by constant (positive) factor.
+ *
+ * Uncertainties are divided by the same factor, such that the *relative*
+ * uncertainties remain unchanged. */
+Histogram operator/(Histogram histogram, const double factor) {
+  return histogram *= 1.0 / factor;
+}
+
```

Comment:
👍  Good that you remembered to do that. Adding the non-member functions that is. Very important.